python/whatsapp_with_vonage/src/main.py

```diff
@@ -0,0 +1,64 @@
+from .utils import get_static_file, throw_if_missing
+from os import environ
```

Comment:
Small nitpick: in other templates we've used just `import os` and `os.environ["ENVAR"]`
Can we match that?